client/src/main/java/io/pravega/client/connection/impl/CommandEncoder.java

```diff
@@ -14,12 +14,9 @@
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.ByteBufOutputStream;
 import io.netty.buffer.Unpooled;
+import io.pravega.common.Exceptions;
 import io.pravega.shared.metrics.MetricNotifier;
-import io.pravega.shared.protocol.netty.Append;
-import io.pravega.shared.protocol.netty.AppendBatchSizeTracker;
-import io.pravega.shared.protocol.netty.InvalidMessageException;
-import io.pravega.shared.protocol.netty.WireCommand;
-import io.pravega.shared.protocol.netty.WireCommandType;
+import io.pravega.shared.protocol.netty.*;
```

Comment:
This is not permitted by checkstyle.

Reply:
Thanks! My IDE automatically did this, I didn't notice that.